server/gc/src/main/java/org/apache/accumulo/gc/replication/CloseWriteAheadLogReferences.java

```diff
@@ -129,12 +129,10 @@ public void run() {
   /**
    * Construct the set of referenced WALs from zookeeper
    *
-   * @param conn
-   *          Connector
    * @return The Set of WALs that are referenced in the metadata table
    */
-  protected HashSet<String> getClosedLogs(Connector conn) {
-    WalStateManager wals = new WalStateManager(conn.getInstance(), ZooReaderWriter.getInstance());
+  protected HashSet<String> getClosedLogs() {
+    WalStateManager wals = new WalStateManager(context, ZooReaderWriter.getInstance());
```

Comment:
This looks right, but be careful here. Replication code has two "clients", and we need to make sure that if we're replacing any client API with an internal type, we're doing so with the "local" system, and not the client code which refers to the remote replica.

Reply:
I checked and this code is OK. It's using the local system